jupyterlab_wipp_plugin_creator/README.md

```diff
@@ -0,0 +1,136 @@
+# jupyterlab_wipp_plugin_creator
+
+<!-- ![Github Actions Status](https://github.com/Kingstonshaw/jupyterlab_wipp_plugin_creator/workflows/Build/badge.svg) -->
+
+<!-- Create wipp plugin by containerizing local code in various languages,  automate the Plugin generation and testing process using both static analysis and templates.  -->
+Enter your plugin information, select the code you wish to containerize, create WIPP plugin automatically.
+
+Current features:
+
+- Right click on file 'Add to WIPP' to mark code to mark code for containerization.
+- On click of 'Create Plugin' button, Post API request containing user input is sent over to the backend.
+- Session persistent frontend database to store paths to marked files using IStateDB.
+- Create plugin.json, dockerfile, requirements.txt based on inputs.
+- Register plugin automatically on WIPP CI https://wipp-ui.ci.aws.labshare.org/plugins.
+- Create temp staging folder with random ID and copy selected codes inside.
+
+This extension is composed of a Python package named `jupyterlab_wipp_plugin_creator`
+for the server extension and a NPM package named `jupyterlab_wipp_plugin_creator`
+for the frontend extension.
+
+
+## Requirements
+
+* JupyterLab >= 3.0
+
+## Install
+
+To install the extension, execute:
+
+```bash
+pip install jupyterlab_wipp_plugin_creator
+export WIPP_UI_URL="https://wipp-ui.ci.aws.labshare.org"
+export WIPP_API_INTERNAL_URL="http://wipp-ui.ci.aws.labshare.org/api/"
+export WIPP_NOTEBOOKS_PATH="/home/jovyan/notebooks/"
+export PLUGIN_TEMP_LOCATION= must enter here your choice of temprorary folder
+```
+
+The last steps are required environment variables from the terminal if not using from CI. Choose a local temporary directory where you want your manifest and code of your plugin to be staged.
+
+- `WIPP_UI_URL` is the WIPP frontend base URL which will be used to open collection pages.
```

Comment:
Currently `WIPP_UI_URL` is mentioned in the export above, which I replaced the URL with place holders, should I just remove it and others from export since the server has it already then?

Reply:
`WIPP_UI_URL` should be removed from the instructions for this extension, because it is not currently being used by this extension.

I think it will be used in the future to give a link to newly created extension, but currently it has no use, so better to remove